Cookbook/Technical-Documents/AutomationIdentifiers.md

```diff
@@ -1,76 +1,85 @@
 
 ## Updating Locators
 
 The quick solution is to simply update the enum. When an identifier's text has changed, this is very straight forward and simple. Update the text defined in the enum to the correct value and automation should start working. Resulting in Pull Requests like this https://github.com/Babylonpartners/babylon-ios/pull/6475
 
 ```swift
-	fileprivate enum NavigationBar {
-			static let header = "Privacy policy"
-			- static let telus = "Babylon by TELUS Health Privacy Policy"
-			+ static let telus = "Privacy Policy"
-	 }
+fileprivate enum NavigationBar {
+    static let header = "Privacy policy"
+    - static let telus = "Babylon by TELUS Health Privacy Policy"
+    + static let telus = "Privacy Policy"
+}
 ```
 
-While this will fix the test, it is susceptible to future changes. In this case time permitting, we should attempt to define a accessibilityIdentifier for the element and then update the enum to use this identifier. This is not always possible as is the case with chat responses, but it is something we should strive for in order to make our testings more stable. And at the same time reducing the amount of updates needed in the future.
+While this will fix the test, it is susceptible to future changes. In this case time permitting, we should attempt to define an `accessibilityIdentifier` for the element and then update the enum to use this identifier. This is not always possible as in the case with chat responses, but it is something we should strive for in order to make our tests more stable and maintainable, reducing the amount of updates needed in the future.
 
 #### Updating Queries
 
-The most common root cause is that the **XCUIElementQuery** requires a update. The thing we need to consider is that the performance of the framework is effected by how much of the view hierarchy is searched when looking for an element. This is because the framework will by default search the entire view hierarchy. This is time consuming, the recommended solution is to give the query as much information as possible. For example, below, we are telling the framework that the the button is within an alert, this improves the performance of the test.
+The most common root cause is that the **XCUIElementQuery** requires an update. The thing we need to consider is that the performance of the XCTest framework is affected by how much of the view hierarchy is searched when looking for an element. This is because the framework will by default search the entire view hierarchy. This is time consuming, the recommended solution is to give the query as much information as possible. For example, below, we are telling the framework that the button is within an alert, this improves the performance of the test.
```

Comment:
```suggestion
The most common root cause is that the `XCUIElementQuery` requires an update. The thing we need to consider is that the performance of the XCTest framework is affected by how much of the view hierarchy is searched when looking for an element. This is because the framework will by default search the entire view hierarchy. This is time consuming, the recommended solution is to give the query as much information as possible. For example, below, we are telling the framework that the button is within an alert, this improves the performance of the test.
```